pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -969,7 +990,7 @@ void runCmd() throws Exception {
     @Parameters(commandDescription = "Update a Pulsar Function that has been deployed to a Pulsar cluster")
     class UpdateFunction extends FunctionDetailsCommand {
 
-        @Parameter(names = "--update-auth-data", description = "Whether or not to update the auth data")
+        @Parameter(names = "--update-auth-data", description = "Whether or not to update the auth data #Java,Python")
```

Comment:
Why not add a new parameter? Parsing string literals is generally error prone, though.

Reply:
Indeed, but AFAIK there's no way to add a new field here. Please see [JCommander Docs](https://jcommander.org) for more info.

Reply:
@SignorMercurio thanks for your information! It seems a JCommander limitation. Then I'm OK with current approach, while we can possibly use another annotation to avoid string parsing but it's not a blocker to this patch.